src/amp-events.js

```diff
@@ -15,17 +15,19 @@
  */
 
 /**
-* Enum used to specify custom Amp Events
-*
-* @enum {string}
-*/
+ * Common AMP events.
+ * @enum {string}
+ */
 export const AmpEvents = {
-  VISIBILITY_CHANGE: 'amp:visibilitychange',
-  DOM_UPDATE: 'amp:dom-update',
-  BUILT: 'amp:built',
-  ATTACHED: 'amp:attached',
-  STUBBED: 'amp:stubbed',
-  LOAD_START: 'amp:load:start',
-  LOAD_END: 'amp:load:end',
-  ERROR: 'amp:error',
+  // Use short, unique strings to reduce bundle size impact.
+  BUILT: 'e1',
+  DOM_UPDATE: 'e2',
+  // The following codes are only used for testing.
+  // TODO(choumx): Move these to a separate enum so they can be DCE'd.
+  ATTACHED: 'e3',
+  STUBBED: 'e4',
+  LOAD_START: 'e5',
+  LOAD_END: 'e6',
+  ERROR: 'e7',
+  VISIBILITY_CHANGE: 'e8',
```

Comment:
Ads uses this one.

Reply:
I only saw usage in test code but upon closer inspection it's actually externally visible. 🤦‍♂️ 

> Additionally, one can observe the amp:visibilitychange on the window object to be notified about changes in visibility.

https://github.com/ampproject/amphtml/blob/master/ads/README.md#page-visibility

Updated the unit test to use the string literal to catch this.